distributed/client.py

```diff
@@ -333,6 +333,9 @@ async def _result(self, raiseit=True):
         await self._state.wait()
         if self.status == "error":
             exc = clean_exception(self._state.exception, self._state.traceback)
+            logger.error(
+                f"in task: {self._state.key}\non worker:{self._state.erred_on}"
+            )
```

Comment:
This will look really weird if anything captures the log. Like most production clients will.